u_prng.go

```diff
@@ -69,68 +61,36 @@ func newPRNG() (*prng, error) {
 	if err != nil {
 		return nil, err
 	}
-	return newPRNGWithSeed(seed), nil
+	return newPRNGWithSeed(seed)
 }
 
 // newPRNGWithSeed initializes a new PRNG using an existing seed.
-func newPRNGWithSeed(seed *PRNGSeed) *prng {
+func newPRNGWithSeed(seed *PRNGSeed) (*prng, error) {
+	shake := sha3.NewShake256()
```

Comment:
Would Shake128 be enough? I have no objections against 256, but some uTLS users seemed to be concerned about performance.

Reply:
This comment about image space is probably relevant:

> if your image space (e.g. the permutations of extensions/cipher suites/etc) exceeds 2^64 by a large amount...

https://github.com/refraction-networking/utls/pull/20#discussion_r258316476

We have a 256-bit seed, so sticking with 256-bit security strength (https://godoc.org/golang.org/x/crypto/sha3#NewShake256) makes sense, I think.